src/Handler/Workspace/Create/CreateWorkspaceHandler.php

```diff
@@ -111,6 +109,9 @@ public function __invoke(
             $this->waitUntilBindingsPropagate($cloudResourceManager, $projectName, $wsServiceAcc->getEmail());
         });
 
+        // generate credentials
+        [$privateKey, $publicPart] = $iamService->createKeyFileCredentials($wsServiceAcc);
+
```

Comment:
sem to předtím posunul na dřív, řešil sem něco kvůli retry, ale to není potřeba a asi to dělá bordel, takže dávám tak pozdě jak to jen jde.
Su zvědavý na čem to bude padat teď.